packages/web-components/src/tabs/tabs.styles.ts

```diff
@@ -0,0 +1,258 @@
+import { css } from '@microsoft/fast-element';
+import {
+  borderRadiusCircular,
+  borderRadiusMedium,
+  colorNeutralForeground1,
+  colorNeutralForeground1Hover,
+  colorNeutralForeground2,
+  colorNeutralForegroundDisabled,
+  colorSubtleBackgroundHover,
+  colorSubtleBackgroundPressed,
+  curveDecelerateMax,
+  durationSlow,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontSizeBase400,
+  fontWeightRegular,
+  lineHeightBase300,
+  lineHeightBase400,
+  spacingHorizontalMNudge,
+  spacingHorizontalSNudge,
+  spacingVerticalL,
+  spacingVerticalMNudge,
+  spacingVerticalS,
+  spacingVerticalSNudge,
+  spacingVerticalXXS,
+  strokeWidthThicker,
+} from '../theme/design-tokens.js';
+import { TabTokenNames } from './tabs.js';
+
+export const styles = css`
+  :host {
+    display: grid;
+    box-sizing: border-box;
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    line-height: ${lineHeightBase300};
+    color: ${colorNeutralForeground2};
+    grid-template-columns: auto 1fr auto;
+    grid-template-rows: auto 1fr;
+  }
+
+  :host([disabled]) {
+    cursor: not-allowed;
+    color: ${colorNeutralForegroundDisabled} !important;
+  }
+
+  :host([disabled]) ::slotted(fluent-tab) {
+    pointer-events: none;
+    cursor: not-allowed;
+    color: ${colorNeutralForegroundDisabled} !important;
+  }
+  :host([disabled]) ::slotted(fluent-tab:after) {
+    background-color: ${colorNeutralForegroundDisabled};
+  }
+  :host([disabled]) ::slotted(fluent-tab[aria-selected='true'])::after {
+    background-color: ${colorNeutralForegroundDisabled};
+  }
+  :host([disabled]) ::slotted(fluent-tab:hover):before {
+    content: unset;
+  }
+
+  .tablist {
+    display: grid;
+    grid-template-rows: auto auto;
+    grid-template-columns: auto;
+    position: relative;
+    width: max-content;
+    align-self: end;
+    box-sizing: border-box;
+  }
+  ::slotted([slot='start']),
+  ::slotted([slot='end']) {
+    display: flex;
+    align-self: center;
+  }
+  ::slotted([slot='start']) {
+    margin-inline-end: 11px;
+  }
+  ::slotted([slot='end']) {
+    margin-inline-start: 11px;
+  }
+
+  // TODO: remove active-indicator styling when FAST Pr to remove active indicator is completed.
+  .active-indicator {
```

Comment:
there is an Active indicator is an attribute you can just st to false, actually @chrisdholt Haven't we already removed this?

Reply:
You are right, it seems the .active-indicator style is not necessary. Removed!